doc/nanocourse/bwt.md

```diff
@@ -367,3 +367,42 @@ Here is the full solution in case you need it:
 \include test/snippet/nanocourse/backward_search.cpp
 
 \endsolution
+
+# Real data example with SeqAn3
+
+Follow the [SeqAn3 Setup tutorial](https://docs.seqan.de/seqan/3-master-user/setup.html) and add an executable
+`count_occurrences` to the `CMakeLists.txt`.
+
+Create a new file `count_occurrences.cpp` in the `source` directory with the following content:
+
+```cpp
+#include <seqan3/io/sequence_file/all.hpp>
+#include <seqan3/search/all.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+    sequence_file_input reference_in{"../reference.fasta.gz"};
+    sequence_file_input query_in{"../reads.fasta.gz", fields<field::SEQ>{}};
+
+    fm_index index{get<field::SEQ>(*reference_in.begin())};
+
+    size_t counter{0u};
+
+    for (auto & [seq] : query_in)
+        counter += std::ranges::size(search(seq, index)) > 100;
+
+    std::cout << "Found " << counter << " queries with more than 100 exact hits.\n";
+}
+```
+
+Now download the
+[reference file](http://ftp.imp.fu-berlin.de/pub/SeqAn/learning_resources/nanocourse/reference.fasta.gz) and
+[read file](http://ftp.imp.fu-berlin.de/pub/SeqAn/learning_resources/nanocourse/reads.fasta.gz) into the `tutorial`
+directory.
+
+Run `cmake -DCMAKE_CXX_FLAGS="-O3 -march=native" ../source` within the `build` directory and execute the program via
+`./count_occurrences`.
+
+Addtionally, measure runtime and memory consumption via `/usr/bin/time -v ./count_occurrences`.
```

Comment:
Add something like:

The above code basically does the same things as you just implemented in this tutorial.
The SeqAn library provides a lot of commonly needed functionality when working with high throughput sequencing,
such that you do not need to implement your own data structures and algorithms.... blubb